src/System.Runtime/tests/System/Lazy.cs

```diff
@@ -82,98 +126,95 @@ public static void TestValueExceptions()
 
```

Comment:
It'd be better if all of this stuff wasn't in the Throws delegate; the only thing that should be in the Throws is the call to Value